.github/workflows/typechecks.yml

```diff
@@ -0,0 +1,43 @@
+name: Tests, Linter & Typecheck
+
+on: pull_request
+
+jobs:  
+  lint:
+    name: 'Linting Markdown'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Use Node.js 16.7
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.7
+
+      - run: npm install -g markdownlint-cli@0.23.2
+      - run: markdownlint '**/*.md' --ignore node_modules
+  test:
+    name: 'Typechecking'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Use Node.js 16.7
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.7
+
+      - name: Install NPM dependencies
+        run: npm install
+      
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
```

Comment:
Didn't we land land on using this instead? https://github.com/cloudflare/wrangler2/blob/main/.github/workflows/prereleases.yml#L24-L30

Reply:
Yes, we did, I missed updating this one.